README.md

```diff
@@ -1,45 +1,51 @@
 # Engineering Resources
+
 List of resources and project samples for Engineering Students
 
 ## Contents
+
 - [Engineering Resources](#engineering-resources)
   - [Contents](#contents)
   - [Introduction](#introduction)
   - [Student Offers <a name="student-programs"></a>](#student-offers-)
   - [CS Resources <a name="education"></a>](#cs-resources-)
   - [Machine Learning Resources <a name="machine-learning"></a>](#machine-learning-resources-)
   - [Competitive Programming <a name="competitive-programming"></a>](#competitive-programming-)
-  
```

Comment:
Add web development section here and make sure it is pointing to the added section as well